.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
-*       @ahamidi
+# Define code owners (individuals or teams that are responsible for code in this repository)
+# More about code owners at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+*       @ConduitIO/conduit-core
```

Comment:
Once this is moved to a repo under conduitio, conduit-core should be automatically asked for a review, which is I added this. It will be removed by the setup script, and I believe the link to the docs about code owners is also sufficient (i.e. an example is not a must have in this case).